.eslintrc

```diff
@@ -2,5 +2,8 @@
 	"env": {
 		"node": true,
 		"es6": true
+	},
```

Comment:
```suggestion
	},
      extends: [ 'airbnb-base' ],
```

Reply:
The problem is that if I add `airbnb` code checks it finds 137 errors.  I don't know if there is any value now in re-writing woking code files to resolve style problems.

Reply:
ok, just leave as is